lib/master.js

```diff
@@ -257,7 +257,8 @@ Master.prototype._startWorkers = function(remainingWorkers, res) {
                         worker.removeListener('exit', workerExit);
                         self._currentStartingWorker = undefined;
                         self._firstWorkerStarted = true;
-                        res.push(msg.serviceReturns[0]);
+                        res.push(msg.serviceReturns.length > 1 ?
+                            msg.serviceReturns : msg.serviceReturns[0]);
```

Comment:
This would mix return values from services in different workers, rather than keeping them grouped.